include/chemfiles/config.h

```diff
@@ -0,0 +1,34 @@
+// Chemfiles, a modern library for chemistry file reading and writing
```

Comment:
This file should not be committed, it is generated by cmake as needed. Are you building inside the source directory instead of a separate build folder?

Reply:
Sorry this was a mistake, I shouldn't have committed this. I do have a separate build folder but I was trying (in another project) to include chemfiles' sources in order to debug and test.

Reply:
No worries, we should add these files to the gitignore to ensure this can not happen again =)